src/main/kotlin/converter/file/FileContentTypeConverter.kt

```diff
@@ -0,0 +1,15 @@
+package com.htmlbuilder.converter.file
+
+import com.htmlbuilder.converter.ContentTypeConverter
+import com.htmlbuilder.model.dto.ContentType
+
+class FileContentTypeConverter : ContentTypeConverter {
```

Comment:
Suggestion: You can make FileContentTypeCinverter an `object` and ContentTypeConverter a sealed interface